src/containers/Token/index.js

```diff
@@ -67,6 +67,7 @@ class Token extends Component {
     const currency = match.params.currency || '';
     const showError = accountId === prevId && error;
 
+    // TODO: title this so it includes the currency code somehow
     document.title = `${t('xrpl_explorer')} | ${accountId.substr(0, 12)}...`;
```

Comment:
why can't you add the code already?

Reply:
I spent a couple minutes thinking about it and couldn't really come up with a good way to do this for the currency codes that were just hex (aka more than 3 characters).